tools/sim/start_carla.sh

```diff
@@ -2,16 +2,23 @@
 
 # Requires nvidia docker - https://github.com/NVIDIA/nvidia-docker
 if ! $(apt list --installed | grep -q nvidia-container-toolkit); then
-  if [ -z "$INSTALL" ]; then
-    echo "Nvidia docker is required. Re-run with INSTALL=1 to automatically install."
-    exit 0
-  else
+  read -p "Nvidia docker is required. Do you want to install it now? (y/n)";
+  if [ "${REPLY}" == "y" ]; then
     distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
     echo $distribution
     curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
     curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
-    sudo apt-get update && sudo apt-get install -y nvidia-container-toolkit
+    sudo apt-get update && sudo apt-get install -y nvidia-docker2 # Also installs docker-ce and nvidia-container-toolkit
+    read -p "Just installed docker? Set permanent docker permission for usergroup. (Run 'sudo systemctl restart docker' afterwards and then rerun start_carla.sh) (y/n)"
+    if [ "${REPLY}" == "y" ]; then
+        # Adding docker to current usergroup
+        sudo groupadd docker
+        sudo usermod -aG docker $USER
+        newgrp docker
+    fi
```

Comment:
this is very much up to the user, don't think we should include that in this script